Packages/OctoPrint/config.yaml

```diff
@@ -55,6 +55,13 @@ system:
   - action: restart mjpg
     command: sudo systemctl restart mjpg.service
     name: Restart Mjpg
+terminalFilters:
+  - name: Suppress temperature messages
+    regex: '(Send: (N\d+\s+)?M105)|(Recv:\s+(ok\s+)?.*\s(T\d*):\d+)'
+  - name: Suppress SD status messages
+    regex: '(Send: (N\d+\s+)?M27)|(Recv: SD printing byte)|(Recv: Not SD printing)'
```

Comment:
This is not needed - it's part of the default OctoPrint handling for M27 responses afaik. I don't think we need to change it from default, as we've tried to fix Redeem to send proper responses...

Reply:
As I mentioned, the stock OP filters are flawed. As they are written, I cannot use some perfectly valid names without having the filter hide the result. We have the opportunity to "patch" their filter in the configuration file that we provide. That is like we "patch" redeem to match the printer hardware.

Reply:
I must've missed that reference - was it in Slack or a different PR?

Reply:
No. It is just above this dialog.

The "temperature" filter was generating a false positive on M115 ...

Reply:
I was specifically talking about the `M27` regex line here. The `M115` change is coherent.

Reply:
It is no change to that line. I was under the impression that we needed to overwrite the whole set of filter options with our desired set.

Reply:
I have confirmed that if we have a `terminalFilters` section, it MUST include ALL filters that we wish to make available.

Reply:
Can the filters be also enabled by default? I find myself always enabling them.

Reply:
@RaveGun - Like you, I would like to have them "on" by default.  But that is really determined by OctoPrint. If there is a setting for that, we can certainly set it in the config that we pass to OP.

Please create an issue here for it. It would be an enhancement that someone can research.